apps/sms/test/marionette/generators/thread.js

```diff
@@ -59,7 +59,7 @@ var ThreadGenerator = {
 
     for (var i = 0; i < parameters.numberOfMessages; i++) {
       var message = null;
-      var messageBody = thread.body;
+      var messageBody = thread.body + ' (threadId: ' + thread.id + ')';
```

Comment:
this makes it easier to see if the message is in the right thread id when debugging